signalfx-org-metrics/metrics.yaml

```
@@ -2219,3 +2226,14 @@ sf.org.rum.grossContentBytesReceived:
 
   metric_type: counter
   title: sf.org.rum.grossContentBytesReceived
+
+  sf.org.rum.grossReplayContentBytesReceived: NBED
```

looks like an alignment issue below as well

```suggestion
  sf.org.rum.grossReplayContentBytesReceived:
```